diagnostics/src/daq_trace/mod.rs

```diff
@@ -79,6 +79,46 @@ impl DigitiserData {
     }
 }
 
+type SharedData = Arc<Mutex<HashMap<u8, DigitiserData>>>;
```

Comment:
Are the changes to this file intentional?

Reply:
They are not.